lib/streamlit/elements/image_proto.py

```diff
@@ -307,7 +307,10 @@ def image_to_url(
                 image = textfile.read()
         # If it's an SVG string, then format and return an SVG data url
         if image.startswith("<svg") or image.strip().startswith("<svg"):
-            return f"data:image/svg+xml,{image}"
+            image = image.strip()
```

Comment:
I think we can move the image.strip to just once above the if statement since it's potentially being done twice

Reply:
done, and pushed (Y)